docs/node-developers/codebase-overview.mdx

```diff
@@ -41,17 +41,17 @@ One difficult issue with learning OCaml is locating and reading documentation fo
     Unix  <-  Core  ->       Async
 ```
 
-There are actually multiple sources for `Core` documentation. To find the correct documentation, first go the GitHub then use that to locate the correct documentation. If you can't find the module you're looking for go to `Core_kernel` and if that fails then look for `Base` also note that find will not work if the sections are not expanded.
+There are multiple sources for OCaml Core library documentation. In general, the source code of an installed library is not available so to find the  documentation, first go to the codebase on GitHub and then locate the correct documentation. If you don't see the module you're looking for, go to `Core_kernel`. If that fails, then look for `Base`. To use the find capability in GitHub, expand the sections.
```

Comment:
@jasongitmail  what core are we talking about? If it is our codebase, let's be more direct. 

I wasn't clear which GitHub repo we are sending readers to... teach me

Reply:
This is all old content and protocol related, which is not my area, so I'm almost as unfamiliar. 

But our protocol is written in OCaml and it sounds like Protocol Team has something they refer to as `Core`. 

So my suggestion is to avoid the specific phrasing of "OCaml Core library" because that could be interpreted as the [OCaml](https://ocaml.org/) language core, which didn't seem like the original intention.

Reply:
follow up work required, see https://github.com/o1-labs/docs2/issues/339